python/ribasim/ribasim/input_base.py

```diff
@@ -600,12 +598,12 @@ def _write_geopackage(self, path: Path) -> None:
 
 
 class ChildModel(BaseModel):
-    _parent: Any | None = None
+    _parent: "Model | None" = None
```

Comment:
This is not a model perse, it should be a base model.

Reply:
Yeah (and then remove the if TYPE_CHECKING part). If that doesn't work, I'll work on this branch.